mxd/backend-service/src/main/java/org/eclipse/tractusx/mxd/backendservice/service/HttpConnectionService.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ ******************************************************************************/
+
+package org.eclipse.tractusx.mxd.backendservice.service;
+
+import okhttp3.Request;
+import org.eclipse.edc.spi.http.EdcHttpClient;
+import org.eclipse.tractusx.mxd.backendservice.entity.Transfer;
+
+public class HttpConnectionService {
+
+
+    private EdcHttpClient httpClient;
+
+    public HttpConnectionService(EdcHttpClient httpClient) {
+        this.httpClient = httpClient;
+    }
+
+    public HttpConnectionService() {
+    }
+
+    public String getUrlAssets(Transfer receivedModel) {
+        String res = null;
+        try {
+
+            Request getRequest = new Request.Builder()
+                    .url(receivedModel.getEndpoint())
+                    .header("authKey", receivedModel.getAuthKey())
+                    .header("authCode", receivedModel.getAuthCode())
+                    .get()
+                    .build();
+            var response = httpClient.execute(getRequest);
+            var body = response.body();
+            var resp = body.string();
+            if (response.isSuccessful()) {
+                return resp;
+            }
+
+        } catch (Exception e) {
+
```

Comment:
Add some log here. Log exception

Reply:
I have used monitor to log the exception.